pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -141,44 +141,53 @@ func (q *QueryReconciler) runShowRestorePoints(ctx context.Context, dispatcher v
 	q.VRec.Eventf(q.Vrpq, corev1.EventTypeNormal, events.ShowRestorePointsStarted,
 		"Starting show restore points")
 	start := time.Now()
-	if res, errRun := dispatcher.ShowRestorePoints(ctx, opts...); verrors.IsReconcileAborted(res, errRun) {
-		if errRun != nil {
-			err = vrpqstatus.UpdateConditionAndState(ctx, q.VRec.Client, q.VRec.Log, q.Vrpq,
-				v1.MakeCondition(vapi.Querying, metav1.ConditionFalse, "Failed"), stateFailedQuery)
-			if err != nil {
-				errRun = errors.Join(errRun, err)
-			}
-			return errRun
+	errRun := dispatcher.ShowRestorePoints(ctx, opts...)
+	if errRun != nil {
+		q.VRec.Eventf(q.Vrpq, corev1.EventTypeNormal, events.ShowRestorePointsFailed, "Failed when calling show restore points")
```

Comment:
We should use `corev1.EventTypeWarning` instead of `corev1.EventTypeNormal` because its a failure condition.